rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -341,7 +374,21 @@ type Product: <"Defines the product that is the subject of a tradable product de
         one-of
 
 type Basket extends ProductBase: <"Defines a custom basket by referencing a product identifier and its consituents.">
-    basketConstituent Product (1..*) <"Identifies the constituents of the basket">
+    basketConstituent Product (0..*) <"Identifies the constituents of the basket">
+    [deprecated] // will be replaced by portfolioBasketConstituent, to be consequently be renamed basketConstituent, also the one-of condition will be removed
+    portfolioBasketConstituent BasketConstituent (0..*) <"Identifies the constituents of the basket">
+
+  condition: one-of
+
+type BasketConstituent extends Product :<"Identifies the constituents of the basket">
+ quantity QuantitySchedule (0..*)
```

Comment:
Attributes should also have descriptions. In particular, those descriptions should articulate the rationale for the multiple cardinality.